internal/handlers/handlers.go

```diff
@@ -12,15 +11,13 @@ var Repo *Repository
 
 // Repository is the repository struct
 type Repository struct {
-	DB  storage.DatabaseRepo
-	App *config.AppConfig
+	DB storage.DatabaseRepo
 }
 
-// NewRepo creates a new Repository
-func NewRepo(a *config.AppConfig, db *driver.DB) *Repository {
+// NewRepoWithGORM creates a new repository with GORM
+func NewRepoWithGORM(db *driver.GORMDriver) *Repository {
```

Comment:
Consider looking into passing an abstraction as a parameter here, instead of creating a separate function for each case